validation/standard/config.go

```diff
@@ -42,8 +46,9 @@ type EcotoneGasPriceOracleBounds struct {
 	BaseFeeScalar     Uint32Bounds `toml:"base_fee_scalar"`
 }
 
-type StandardConfigTy struct {
+type ConfigTy struct {
```

Comment:
It stands for "type", I made this more explicit in [3993b4f](https://github.com/ethereum-optimism/superchain-registry/pull/229/commits/3993b4f962d26ef24dcdfa8637cfcdf012a5005b). Because I want to export a singleton variable caleld `Config`, I had to call this something else. Otherwise no special meaning.